.travis.yml

```diff
@@ -3,10 +3,17 @@ language: php
 php:
     - 5.5
     - 5.6
+    - 7.0
     - hhvm
 
+matrix:
+    include:
+        - php: 5.5
+          env: 'COMPOSER_FLAGS="--prefer-stable --prefer-lowest"'
+
 before_script:
-    - travis_retry composer install --no-interaction
+    - travis_retry composer self-update
```

Comment:
What's the problem with it?

The reason I usually leave it in is that travis rarely update the composer executable. I had several issues when the `^` was introduced.